spec/controllers/corporate_controller_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe CorporateController, type: :controller do
+RSpec.describe CorporateController, type: :controller, features: [:corporate] do
```

Comment:
I'm certain you don't need to wrap `:corporate` in an array. They get normalised in the library.